package.json

```diff
@@ -1,19 +1,29 @@
-{ "name" : "oauth"
-, "description" : "Library for interacting with OAuth 1.0, 1.0A, 2 and Echo.  Provides simplified client access and allows for construction of more complex apis and OAuth providers."
-, "version" : "0.9.12"
-, "directories" : { "lib" : "./lib" }
-, "main" : "index.js"
-, "author" : "Ciaran Jessup <ciaranj@gmail.com>"
-, "repository" : { "type":"git", "url":"http://github.com/ciaranj/node-oauth.git" }
-, "devDependencies": {
+{
+  "name": "oauth",
+  "description": "Library for interacting with OAuth 1.0, 1.0A, 2 and Echo.  Provides simplified client access and allows for construction of more complex apis and OAuth providers.",
+  "version": "0.9.12",
+  "directories": {
+    "lib": "./lib"
+  },
+  "main": "index.js",
+  "author": "Ciaran Jessup <ciaranj@gmail.com>",
+  "repository": {
+    "type": "git",
+    "url": "http://github.com/ciaranj/node-oauth.git"
+  },
+  "devDependencies": {
     "vows": "0.5.x"
-  }
-, "scripts": {
+  },
+  "scripts": {
     "test": "make test"
-  }
-, "licenses" :
-  [ { "type" : "MIT"
-    , "url" : "http://github.com/ciaranj/node-oauth/raw/master/LICENSE"
+  },
+  "licenses": [
+    {
+      "type": "MIT",
+      "url": "http://github.com/ciaranj/node-oauth/raw/master/LICENSE"
     }
-  ]
+  ],
+  "dependencies": {
+    "request": "^2.54.0"
```

Comment:
usually I wouldn't change the format of a file, but this was done by npm, when doing a `npm install request --save`, so figured it would be better to keep it than only add `dependencies` to the current format.